spec/presenters/keyword_presenter_spec.rb

```diff
@@ -0,0 +1,119 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe KeywordPresenter, type: :presenter do
+  describe '#result_link_types' do
+    it 'returns a list grouped by link_types' do
+      presenter = described_class.new(Fabricate(:keyword_parsed_with_links))
```

Comment:
How about to use non persisted object case in this case?(other test cases also in this file) It will speed up test suite
```suggestion
      presenter = described_class.new(Fabricate.build(:keyword_parsed_with_links))
```

Reply:
Nice, very good to know this one!!
Issue #82 added for later as it is not a blocking point.

Thanks @junan